sdk/identity/identity/CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ## 1.2.4-beta.2 (Unreleased)
 
-- Breaking change: `DefaultAzureCredential` now only uses `InteractiveBrowserCredential` in the browser since it's the only credential intended to be used to authenticate within browsers.
+- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were already not supported for the browser.
```

Comment:
That is true! Thank you 👍

Reply:
Actually! That is not true. The released DefaultAzureCredential throws on the constructor, because it invoked the constructor of credentials that throw in the constructor.

Reply:
Ah! Got it, I wrongly assumed it was the other way around.

Reply:
In that case, I would update the changelog statement to be clear that errors would get thrown both before and after. Only the location from where it was thrown has changed. Something like this:

> The constructor for `DefaultAzureCredential` will throw the error on not supporting browser directly instead of depending on the chained set of credentials to do the same.